sympy/polys/matrices/domainmatrix.py

```diff
@@ -21,6 +21,37 @@
 
 
 class DomainMatrix:
+    r"""
+
+        The idea behind DomainMatrix is to associate :py:class:`~.Domain` with matrix,
```

Comment:
The indentation in the docstring should match that of the docstring itself.

Reply:
Sure, my bad.